firedrake/utility_meshes.py

```diff
@@ -644,6 +644,57 @@ def CircleManifoldMesh(ncells, radius=1, distribution_parameters=None, comm=COMM
     return m
 
 
+def UnitDiskMesh(refinement_level=0, distribution_parameters=None, comm=COMM_WORLD):
+    """Generate a mesh of the unit disk in 2D
+
+    :kwarg refinement_level: optional number of refinements (0 is a diamond)
+    :kwarg reorder: (optional), should the mesh be reordered?
```

Comment:
I don't think the flag takes a `reorder` flag?

Reply:
Woops, I was using the same docstring from other functions in the same module. Just added a reorder flag.